pallets/liquidity-pools/src/message.rs

```diff
@@ -218,8 +220,6 @@ where
 		/// `DecreaseInvestOrder` message, i.e., the amount by which the
 		/// investment order was actually decreased by.
 		currency_payout: Balance,
-		/// The outstanding order, in `currency` units
-		remaining_invest_order: Balance,
```

Comment:
NOTE: This was discussed, see [DAO slack comment](https://centrifugedao.slack.com/archives/C04HCK98KLL/p1692046938307889?thread_ts=1691760753.515679&cid=C04HCK98KLL).